test/models/patient_test.rb

```diff
@@ -78,6 +81,31 @@ class PatientTest < ActiveSupport::TestCase
     end
   end
 
+  describe 'pledge_summary' do
+    it "should return sent and remaining pledge for the next 7 days" do
+      # Doesn't error when there are no pregnancies
```

Comment:
This is kind of a separate feature of this method, no? Might be worth chucking in its own `it` block -- something like:

```
it 'should not choke if there are no pregnancies that week' do
  # destroy all existing patients and their embedded pregnancies
  assert_equal ...
end
```
 maybe?